kcidb/db/bigquery/__init__.py

```diff
@@ -133,8 +134,8 @@ def init(self):
                 f" FROM `{table_ref}` GROUP BY id"
             self.client.create_table(view)
         # Set dataset schema version
-        self.dataset.labels["version_major"] = str(io.schema.LATEST.major)
-        self.dataset.labels["version_minor"] = str(io.schema.LATEST.minor)
+        self.dataset.labels["version_major"] = str(IO_LATEST_VERSION.major)
+        self.dataset.labels["version_minor"] = str(IO_LATEST_VERSION.minor)
```

Comment:
These and similar edits in this file would be incorrect, because we will not necessarily be using the latest schema in the driver, just the one all of kcidb is currently using. So, these should be changed to just `io.SCHEMA` directly.